test/testcafe/spec/ChallengeOktaVerifyFastPassView_spec.js

```diff
@@ -381,6 +381,7 @@ async function setupLoopbackFallback(t, widgetOptions) {
 test
   .meta('gen3', false) // Gen3 does not have the same redundant polling issue as Gen2 and does not need to implement enhancedPollingEnabled, so skip this test
   .requestHooks(loopbackRedundantPollingLogger, loopbackRedundantPollingMock)('in loopback server, redundant polling exists if server returns enhancedPollingEnabled as false', async t => {
+    loopbackRedundantPollingMockPollCount = userVariables.gen3 ? -1 : 0;
```

Comment:
Need to reset vars in order to let test retry in case of failing

Reply:
Could all of this counter reset code be put into the setup code?

Reply:
Updated test. See https://github.com/okta/okta-signin-widget/pull/3714/files#diff-698534168a87ff567c25b350509b7261a3660769a3c5fd36b10adc3873731f39